modules/@apostrophecms/modal/ui/apos/components/ApostropheModal.vue

```diff
@@ -63,7 +63,6 @@ export default {
   border-radius: 2px;
   box-shadow: 0 2px 8px rgba(0, 0, 0, 0.33);
   transition: all 0.3s ease;
-  font-family: map-get($font-families, default);
```

Comment:
This whole file was deleted, so you'll have a little merge conflict. Pretty easy to resolve.